guide/payments/contacts.md

```diff
@@ -0,0 +1,206 @@
+imagesHome:
+    - file: sending-home
+      alt: Home screen with amount input
+      caption: The user enters an amount and taps "Pay".
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: Next, the user is asked to choose a contact to send to.
+    - file: sending-review
+      alt: Invoice screen with a contact assigned
+      caption: Final review of the payment before sending it.
+imagesReceive:
+    - file: receive-home
+      alt: Home screen with amount input
+      caption: The user enters an amount and taps "Request".
+    - file: receive-qr
+      alt: Payment request screen
+      caption: An invoice screen with minimal information.
+    - file: receive-qr-with-contact
+      alt: Payment request screen with a contact assigned
+      caption: The same invoice with a contact assigned.
+imagesActivity:
+    - file: activity
+      alt: Activity screen showing transactions with minimal information
+      caption: By default, payments contain little to no contextual information.
+    - file: activity-annotated
+      alt: Activity screen showing transactions with assigned contacts
+      caption: Assigning contacts to payments makes them much easier to understand and work with.
+    - file: activity-transaction
+      alt: Transaction details
+      caption: Transaction detail screens should have convenient options to assign a contact.
+    - file: sending-pick-contact
+      alt: Screen with a list of contacts to choose from
+      caption: The user picks a contact in a slide-up modal.
+    - file: activity-transaction-annotated
+      alt: Transaction details with assigned contact
+      caption: A payment with an assigned contact.
+
+---
+
+{% include picture.html
+    image = "/assets/images/guide/payments/contacts/contacts-header.jpg"
+    retina = "/assets/images/guide/payments/contacts/contacts-header@2x.jpg"
+    mobile = "/assets/images/guide/payments/contacts/contacts-header-mobile.jpg"
+    mobileRetina = "/assets/images/guide/payments/contacts/contacts-header-mobile@2x.jpg"
+    alt-text = "Illustration of large contact cards"
+    width = 1600
+    height = 400
+    layout = "full-width"
+%}
+
+
+# Contacts
+
+<!--
+
+
+-->
+
+Whether we’re sending emails, physical mail, or following someone on social media, we primarily think in terms of names and faces, and not the respective address or user ID.
+
+Invoices, node IDs and other transaction endpoints in bitcoin and Lightning are highly unintuitive. Abstracting them via a contact list can create a much smoother user experience. There are many [payment request formats]({{ '/guide/payments/request/payment-request-formats/' | relative_url }}), each with unique properties and varying levels of maturity and adoption, requiring unique design solutions. This page uses the more approachable term "address", along with various UI techniques, to abstract these complexities for users.
```

Comment:
I don't see a problem with the #671 page, as it talks about invoices. My UI proposal does not mix invoices with addresses.

It takes some squinting to accept the term "address", for sure. Just like with "street address", "web address", and "email address". There's also "web address" and "IP address", which are pretty different things, but you can type both in the same input field in the browser and get where you want to. That makes "address" a very flexible, and overloaded, term. I just think we have to forget about technical lingo here. Can I repeatedly send something to this weird-looking identifier? Then it's an address (no matter if it's a Lightning address, zPub, offer, etc).

Destination is not quite right, as I can also receive from certain payment request formats. If I save your Lightning address, and you send from that node, it should also be linked to that payment. I know it's not 100% the same thing, as the Lightning address is not baked into the invoice, but it's the same from the user perspective.

This is such a squishy topic that can easily result in totally different outcomes. When I looked at it in the [work stream](https://youtu.be/GPewV_C-w2Q) last week, I basically decided to declare bankruptcy on the technical terminology and trying to find the perfect term, and just go with something that is widely accepted online. Just had to squint. 🤷🏻‍♀️

Reply:
Hmm. Ok, that argument is somewhat convincing to me. 

It looks very clean in your UI designs which simply call it "address", and then the user can expand to see the different formats. I'm curious what happens if using the word in this manner begins to seep out into regular usage. Do people start appending the term "bitcoin address" onto it? They likely would, in order to differentiate from "web address", "email address", etc. Not technically accurate, but I can live with it. We can easily use "on-chain address" to clarify in situations where a bitcoin address is needed.

It would be good to get some other opinions on it, but I'm warming up to the idea.

Reply:
Mmh yeah I think it's starting to warm up on me, I don't see any glaring issues with it! Makes a lot of sense throughout the section